webapp/src/plugin.jsx

```diff
@@ -192,6 +194,10 @@ export default class DemoPlugin {
         registry.registerAdminConsoleCustomSetting('SecretMessage', SecretMessageSetting, {showTitle: true});
         registry.registerAdminConsoleCustomSetting('CustomSetting', CustomSetting);
 
+        registry.registerFilePreviewComponent((fileInfo) => {
+            return isEnabled(store.getState()) && fileInfo.extension === 'demo';
```

Comment:
Is it necessary to check if the plugin is enabled? If the plugin is not enabled the webapp should not have custom preview component for that plugin

Reply:
@marianunez I thought the `isEnabled` selector was pulling from the plugin's config in the system console to showcase a toggleable field, but I am mistaken. There is lots of code on the frontend and backend of this repo that is utilizing this value, when at no point should a code path reach a point where it is disabled. I think this is old code when the plugin wasn't auto-deactivated or something?

Will remove this check :+1: